app/views/play/CampaignView.coffee

```diff
@@ -1008,3 +1081,83 @@ module.exports = class CampaignView extends RootView
 
   onClickBrainPopReplayButton: ->
     api.users.resetProgress().then(=> document.location.reload())
+
+  getLevels: () ->
+    return @courseLevelsFake if @courseLevels?
```

Comment:
I'm not sure what this name means, `courseLevelsFake`. What's its purpose exactly?